seed/tests/test_columns.py

```diff
@@ -1088,7 +1088,7 @@ def test_retrieve_db_field_name_from_db_tables(self):
         """These values are the fields that can be used for hashing a property to check if it is the same record."""
         expected = ['address_line_1', 'address_line_2', 'analysis_end_time', 'analysis_start_time',
                     'analysis_state_message', 'block_number', 'building_certification',
-                    'building_count', 'campus', 'city', 'conditioned_floor_area', 'created',
+                    'building_count', 'campus', 'city', 'conditioned_floor_area',
```

Comment:
I'm not sure why `created` and `updated` were expected to be used (if they existed) for hashing a property.

Reply:
Yeah, I don't think we would want those considered when hashing.  That said, since they were previously being returned by `retrieve_db_field_name_for_hash_comparison()` that may mean we need a migration to recompute all hashes.  @nlong thoughts?

Reply:
Interestingly, I would expect that hashes would need to be recomputed because of this line: https://github.com/SEED-platform/seed/blob/develop/seed/data_importer/tasks.py#L1117

But after importing the same file twice under the following circumstances:
1. including `created` and `updated` in COLUMN_EXCLUDE_FIELDS
2. excluding those fields from COLUMN_EXCLUDE_FIELDS

... the hash values were the same and the records were ignored on the second import? I'm planning to investigate more, but this wasn't what I expected.